docs/modules/api/pages/admin_api.adoc

```diff
@@ -156,6 +156,23 @@ curl -k -u cerbos:cerbosAdmin \
     'https://localhost:3592/admin/policy?id=x.yaml&id=y.yaml'
 ----
 
+=== Disable Policies
+
+----
+DELETE /admin/policy?id=policy_id
+----
+
+NOTE: This endpoint requires a mutable storage driver such as xref:configuration:storage.adoc#sqlite3[sqlite3] to be configured.
+
+Issue a DELETE request to the endpoint with the list of IDs (the `id` query parameter can be repeated multiple times) to disable.
+
+----
+curl -k -u cerbos:cerbosAdmin -X DELETE \
+    'https://localhost:3592/admin/policy?id=principal.donald_duck.vdefault&id=derived_roles.my_derived_roles'
+----
+
+NOTE: If the policy being disabled has scope, its descendants must be disabled first. Otherwise, the storage index will
```

Comment:
```suggestion
CAUTION: Scoped policies must have unbroken scope chains. If you're disabling a scoped policy, make sure that its descendant policies are disabled as well. 
```


If a user accidentally breaks a scoped policy chain and then restarts the Cerbos instance, will it refuse to start? If so, they have no choice but to manually edit the database records, right?

Reply:
Logs when a scoped policy is disabled;
```2023-01-31T11:44:38.126+0300    ERROR   cerbos.compiler Failed to recompile     {"id": {}, "error": "missing policy definition {13106758747738567535}: invalid compilation unit"}```

Cerbos doesn't refuse to start, but I think manual editing of the database records is a must.

Reply:
If it still starts, we can use the CLI to fix the issue, can't we? Then that's not a big problem.